src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a person's contact.
+ */
+public class Contact {
+
+    protected final String value;
```

Comment:
Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.

Reply:
Okay, will take note when assigning access modifiers to variable in future.